_posts/1-2-29-PageIndicators.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: doc
+permalink: /docs/ui-toolkit/components/page-indicators
+title: Page Indicators
+section: UI toolkit
+---
+
+# PageIndicators
+
+Renders a horizontally aligned list of dots (page indicators), which can be useful in `HorizontalPager`, `ImageGallery` or `InlineGallery` components
+
+## PageIndicators
+![PageIndicators example]({{ site.baseurl }}/img/ui-toolkit/horizontal-pager/horizontal_pager@2x.png "PageIndicators"){:.docs-component-image}
```

Comment:
This picture is good, although it should be called page_indicators. Also, it's a bit confusing that the same picture is used in `HorizontalPager`. If I use `HorizontalPager`, but I'm not getting these dots, we shouldn't use that image. Instead, we could provide an example image of using HorizontalPager + PageIndicators + some content that I'm actually scrolling.